greedybear/cronjobs/db_backup.py

```diff
@@ -0,0 +1,17 @@
+from django.core import management
+
+from greedybear import settings
+from greedybear.cronjobs.base import Cronjob
+
+
+class DbBackup(Cronjob):
```

Comment:
without adding this cron as a celery task, it would never run

Reply:
Ok Sir.